pwiz_tools/Skyline/Properties/Resources.resx

```diff
@@ -13878,4 +13878,16 @@ If you choose Disable, you can enable Auto-select later with the "Refine &gt; Ad
   <data name="Server_GetKey___anonymous_" xml:space="preserve">
     <value> (anonymous)</value>
   </data>
+  <data name="CommandLine_ExportSpecLib_Failure_attempting_to_save_spectral_library_file__0_" xml:space="preserve">
+    <value>Failure attempting to save spectral library file {0}</value>

```

Comment:
Errors like this during command-line processing are generally expected to begin with "Error: "